src/main/java/org/jenkinsci/plugins/codesonar/CodeSonarPublisher.java

```diff
@@ -111,7 +122,7 @@ public class CodeSonarPublisher extends Recorder implements SimpleBuildStep {
     @DataBoundConstructor
     public CodeSonarPublisher(
             List<Condition> conditions, String protocol, String hubAddress, String projectName, String credentialId,
-            String visibilityFilter
+            String visibilityFilter, String visibilityFilterNewWarnings, String codesonarProjectFile
     ) {
```

Comment:
I think the new parameter, "visibilityFilterNewWarnings" should be renamed to "newWarningsFilter".  In principle, the old parameter "visibilityFilter" would be named "activeWarningsFilter" (or something similar), but I don't think we can (or should) change the name of the old parameter.